presto-product-tests/README.md

```diff
@@ -1,125 +1,136 @@
+name ```default``` is created. It's settings should be adequate for most cases,
+but setting up docker machine to have at least 4GB is required. Memory size can be
+adjusted using the VirtualBox GUI, or new docker machine with the desired memory size
+can be created.
 
-* docker-compose >= 1.6
+* To create virtual machine with name <machine> (required one time):
 
-[https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/).
+    ```
+    docker-machine create -d virtualbox --virtualbox-memory 4096 <machine>
+    ```
 
-```
-pip install docker-compose
-```
+* To set up enviroment to use `<machine>` instead of ```default``` (required at docker bash startup):
 
-> Note that if you are using Mac OS X and installed docker-toolbox to have docker, 
-> then docker-compose should be already installed on your system.
+    ```
+    eval $(docker-machine env <machine>)
+    ```
 
-### Running product tests
 
-Below manual describe how to set up and teardown docker clusters needed to run product tests.
-It also covers actual product tests execution with usage of these clusters.
+## Running tests
 
-> Note that if you using Mac OS X you may need to configure your environment to make docker commands working. 
-> Additionally, please make sure that your docker machine is able to allocate at least 4GB memory.
+Product tests are not run as part of the maven build process. To start them use run following command:
 
 ```
-docker-machine create -d virtualbox --virtualbox-memory 4096 <machine>
-eval $(docker env <machine>)
+presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query
 ```
 
-#### Configuration profiles
-
-Configuration profiles are stored in `presto-product-tests/etc` diretory. There are two such profiles:
-
-**distributed** - consists of single node (pseudo-distributed) hadoop cluster and multiple node presto cluster
- with one coordinator node and at least one worker node. You can select number of workers by below command:
+Possible profiles are:
+- **distributed** - consists of single node (pseudo-distributed) Hadoop cluster and multiple node Presto cluster
+    with one coordinator node and at least one worker node.
+- **singlenode** - consists of single node (pseudo-distributed) Hadoop cluster and one node Presto cluster
 
-    cd presto-product-tests/etc/distributed
-    docker-compose scale presto-worker=<N>
+*Docker container logs can be found in ```/tmp/presto_docker_logs```*
 
-**singlenode** - consists of single node (pseudo-distributed) hadoop cluster and one node presto cluster
+## Debugging tests
 
-#### Execution
+The following steps explain how you can run the product tests from your IDE as regular TestNG tests:
 
-1. Build Presto project (to have presto-server and product tests compiled).
+1. Start the Hadoop container
 
     ```
-    ./mvnw install
+    docker-compose -f presto-product-tests/conf/docker/singlenode/docker-compose.yml up -d hadoop-master
```

Comment:
I think this is just additional noise and there is no strong tendency of including such step to presto docs (eg. in presto/README.md" section "Running the CLI").
Ayway - done.